modules/cold-extraction/ColdDataRetriever.py

```diff
@@ -277,4 +292,4 @@ def run_threaded(job_func):
         time.sleep(1)
     except KeyboardInterrupt:
         check_kill_process()
-        sys.exit(0)
\ No newline at end of file
+        sys.exit(0)
```

Comment:
This change can be omitted as it just changes the new line.


By performing a "git diff" before committing and pushing your changes, you can confirm that you are submitting only the necessary minimal changes to the repository.

Reply:
Ok sure
will follow this